rules/nette/src/Rector/MethodCall/ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Nette\Rector\MethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Identifier;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * @see \Rector\Nette\Tests\Rector\MethodCall\ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector\ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRectorTest
+ */
+final class ConvertAddUploadWithThirdArgumentTrueToAddMultiUploadRector extends AbstractRector
+{
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition('convert addUpload() with 3rd argument true to addMultiUpload()', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+$form = new Nette\Forms\Form();
+$form->addUpload('...', '...', true);
+CODE_SAMPLE
+
+                ,
+                <<<'CODE_SAMPLE'
+$form = new Nette\Forms\Form();
+$form->addMultiUpload('...', '...');
+CODE_SAMPLE
+
+            ),
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isObjectType($node->var, 'Nette\Forms\Form')) {
+            return null;
+        }
+
+        if (! $this->isName($node->name, 'addUpload')) {
+            return null;
+        }
+
+        $args = $node->args;
+        if (! isset($args[2])) {
+            return null;
+        }
+
+        $thirdArgumentValue = $node->args[2]->value;
+        if (
+            $thirdArgumentValue instanceof ConstFetch
+            && count($thirdArgumentValue->name->parts) === 1
+            && $thirdArgumentValue->name->parts[0] === 'true'
```

Comment:
Here could be used `$this->isTrue(...)` method

Reply:
implemented.